appveyor.yml

```diff
@@ -0,0 +1,47 @@
+services:
+  - mssql2016
+
+environment:
+  matrix:
+    - TOXENV: py27-dj18-mssql-wt18
+    - TOXENV: py34-dj18-mssql-wt18
+    - TOXENV: py35-dj18-mssql-wt18
+    - TOXENV: py27-dj110-mssql-wt18
+    - TOXENV: py34-dj110-mssql-wt18
+    - TOXENV: py35-dj110-mssql-wt18
+    - TOXENV: py27-dj18-mssql-wt19
+    - TOXENV: py34-dj18-mssql-wt19
+    - TOXENV: py35-dj18-mssql-wt19
+    - TOXENV: py27-dj110-mssql-wt19
+    - TOXENV: py34-dj110-mssql-wt19
+    - TOXENV: py35-dj110-mssql-wt19
+    - TOXENV: py34-dj18-mssql-wt110
+    - TOXENV: py35-dj18-mssql-wt110
+    - TOXENV: py27-dj110-mssql-wt110
+    - TOXENV: py34-dj110-mssql-wt110
+    - TOXENV: py35-dj110-mssql-wt110
+    - TOXENV: py27-dj111-mssql-wt110
+    - TOXENV: py34-dj111-mssql-wt110
+    - TOXENV: py35-dj111-mssql-wt110
+    - TOXENV: py36-dj111-mssql-wt110
+    - TOXENV: py27-dj18-mssql-wt111
+    - TOXENV: py34-dj18-mssql-wt111
+    - TOXENV: py35-dj18-mssql-wt111
+    - TOXENV: py27-dj110-mssql-wt111
+    - TOXENV: py34-dj110-mssql-wt111
+    - TOXENV: py35-dj110-mssql-wt111
+    - TOXENV: py27-dj111-mssql-wt111
+    - TOXENV: py34-dj111-mssql-wt111
+    - TOXENV: py35-dj111-mssql-wt111
+    - TOXENV: py36-dj111-mssql-wt111
```

Comment:
Probably not but I'm struggling on what the best balance for this would be. Any suggestions?